bip-update-process.md

```diff
@@ -0,0 +1,708 @@
+```
+BIP: ?
+Title: Updated BIP Process
+Author: Murch <murch@murch.one>
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-Updated-BIP-Process
+Status: Draft
+Type: Process
+Created: 2024-05-13
+License: BSD-2-Clause
+Post-History: https://github.com/murchandamus/bips/pull/2
+Requires: 123
+Replaces: 2
+```
+
+## Abstract
+
+This _Bitcoin Improvement Proposal (BIP)_ provides information about the preparation of BIPs and policies relating to
+the publication of BIPs. It replaces [BIP 2](bip-0002.mediawiki) with a streamlined process, and may be amended to
+address the evolving needs of the BIP process.
+
+## Motivation
+
+BIP 2 is over eight years old and was written when different concerns were pressing to the Bitcoin[^capitalization]
+community. The BIP process as defined by BIP 2 thus far seems to have been fashioned to facilitate the design and
+activation of protocol changes. In the past years, BIPs more often describe interoperable features beyond the base
```

Comment:
BIPs involving protocol changes are still being proposed, so I'm not sure about this sentence and if it is necessary.

```suggestion
activation of protocol changes. In the past years, BIPs have more often described interoperational standards beyond the base
```